numpy/ma/tests/test_extras.py

```diff
@@ -1686,3 +1686,103 @@ def test_stack_nd(self):
         assert_equal(c.shape, c_shp)
         assert_array_equal(a1.mask, c[..., 0].mask)
         assert_array_equal(a2.mask, c[..., 1].mask)
+
+class TestSplit:
```

Comment:
The tests here aren't specific to the `ma` versions of these functions. For example, they don't test that the output of `np.ma.split`, `np.ma.hsplit`, etc. are MaskedArrays. The non-ma functionality is already covered in the non-ma version of the tests (e.g. `numpy/lib/tests/test_shape_base.TestSplit`).